packages/flutter_tools/lib/src/commands/make_host_app_editable.dart

```diff
@@ -10,69 +10,57 @@ import '../runner/flutter_command.dart';
 
 class MakeHostAppEditableCommand extends FlutterCommand {
   MakeHostAppEditableCommand() {
-    addSubcommand(MakeHostAppEditableAndroidCommand());
-    addSubcommand(MakeHostAppEditableIosCommand());
+    requiresPubspecYaml();
+
+    argParser.addFlag(
+      'ios',
+      help: 'Whether to make this project\'s iOS app editable.',
+      negatable: false,
+    );
+    argParser.addFlag(
+      'android',
+      help: 'Whether ot make this project\'s Android app editable.',
+      negatable: false,
+    );
   }
 
-  @override
-  final String name = 'make-host-app-editable';
+  FlutterProject _project;
 
   @override
-  final String description = 'Commands for making host apps editable within a Flutter project';
+  final String name = 'make-host-app-editable';
 
   @override
-  bool get hidden => true;
+  final String description = 'Converts generated host apps to editable, non-generated host apps.\n\n'
```

Comment:
This line is a little confusing at first read - maybe something like `'Moves the Flutter controlled and generated host app(s) to a user controlled location.\n\n'`?

Reply:
Tried some different wording